react1/week2/preparation.md

```diff
@@ -1,10 +1,4 @@
 # Preparation
 
-- [Using the State Hook](https://reactjs.org/docs/hooks-state.html) (20 min)
-- [Using the Effect Hook](https://reactjs.org/docs/hooks-effect.html) (20 min)
-- https://github.com/facebook/create-react-app (10 min)
-
-## Flipped classroom videos
-
-- [React 1 week 2 part 1](https://youtu.be/ArGRhOgf5LU)
-- [React 1 week 2 part 2](https://youtu.be/E_5U6lzI8-A)
+- [Using the State Hook](https://react.dev/reference/react/useState) (20 min)
+- https://vercel.com/blog/nextjs-server-side-rendering-vs-static-generation (Optional - 10 min)
```

Comment:
Bit picky but can we use a link label here (e.g. "NextJS Server Side Rendering vs Static Generation") so it looks nice like the other links you've added 😇